apps/src/StudioApp.js

```diff
@@ -1212,11 +1212,11 @@ StudioApp.prototype.inject = function (div, options) {
     customSimpleDialog: this.feedback_.showSimpleDialog.bind(this.feedback_)
   };
 
-  // Allows Google Blockly labs to use the Zelos or Thrasos renderer instead of the default.
+  // Allows Google Blockly labs to use the Zelos or legacy Geras renderer instead of the default Thrasos.
   if (experiments.isEnabled('zelos')) {
     options.renderer = 'cdo_renderer_zelos';
-  } else if (experiments.isEnabled('thrasos')) {
-    options.renderer = 'cdo_renderer_thrasos';
+  } else if (experiments.isEnabled('geras')) {
+    options.renderer = 'cdo_renderer';
```

Comment:
What do you think of renaming `cdo_renderer` to cdo_renderer_geras` and also renaming the class `CdoRenderer` to `CdoRendererGeras`?

Reply:
That sounds good to me. It looks like you’ve already prepped that change in your branch so I’ll merge this one as is.

Reply:
Sounds good - thanks! For documentation: [prepped branch](https://github.com/code-dot-org/code-dot-org/tree/alice/sprite-lab-socket-shapes)